package.json

```
@@ -41,40 +41,40 @@
     "@semantic-release/exec": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/github": "^8.0.2",
-    "@semantic-release/npm": "^8.0.3",
-    "@types/jest": "^27.0.3",
-    "@types/react": "^17.0.37",
-    "dotenv-cli": "^4.1.1",
-    "jest": "^27.4.5",
-    "jest-extended": "^1.2.0",
+    "@semantic-release/npm": "9.0.0",
+    "@types/jest": "27.4.0",
+    "@types/react": "17.0.39",
+    "dotenv-cli": "5.0.0",
+    "jest": "27.5.1",
+    "jest-extended": "2.0.0",
     "react": "^17.0.2",
     "replace-json-property": "^1.8.0",
-    "semantic-release": "^18.0.1",
-    "ts-jest": "^27.1.2",
-    "typescript": "^4.5.4"
+    "semantic-release": "19.0.2",
+    "ts-jest": "27.1.3",
+    "typescript": "4.5.5"
   },
   "dependencies": {
     "@react-native-community/eslint-config": "^3.0.1",
-    "@typescript-eslint/eslint-plugin": "^5.7.0",
-    "@typescript-eslint/parser": "^5.7.0",
-    "eslint": "^8.5.0",
+    "@typescript-eslint/eslint-plugin": "5.11.0",
+    "@typescript-eslint/parser": "5.11.0",
+    "eslint": "8.8.0",
```

Waarom worden geen patches toegelaten?
```suggestion
    "eslint": "~8.8.0",
```

---

Ik ga liefst bewust om met version bumps. Heb in het verleden al packages kapot weten gaan door breaking changes in een "patch version".

Ik ben ongetwijfeld scarred daardoor, maar het is wel "veiliger" in mijn ogen. :-)

---

most libraries don't really post a clear summary of what's in a patch, so not sure how this helps you be concious about versions. ^ is de npm standard to ensure that bugfixes and, more importantly security fixes, flow through the ecosystem as fast as possible. Since you have a CI any issues with building should be caught.
the ~ operator is very cursed though, I don't know why any dev would want to use "approximately equivalent to" haha

---

> "approximately equivalent to" haha

Well `~` means patches only (Expo uses this commonly nowadays)
And `^` means patches and minor versions (more optimistic/dangerous)